projects/timer_assignment/src/main.c

```diff
@@ -0,0 +1,48 @@
+#include <stdint.h>
+#include "interrupt.h"
```

Comment:
Super tiny nitpick: there should be a blank line between standard headers and midsun-specific headers.